src/dependency.rs

```diff
@@ -83,3 +83,5 @@ pub struct Dependency {
 }
 
 pub use cargo_platform::Platform;
+
+use crate::Source;
```

Comment:
😆 I have only myself to blame for the import location

Reply:
Fixed.